cnc_ctrl_v1/System.cpp

```diff
@@ -414,18 +453,28 @@ int getPCBVersion(){
             pinCheck &= B000011; // strip off the unstrapped bits
             TLE5206 = false;
             TLE9201 = false;
+            TB6643 = false;
             break;
         case B110100: case B000100: // some versions of board v1.4 don't strap VERS5-6 low
             pinCheck &= B000111;    // strip off the unstrapped bits
+            TB6643 = false;            
             TLE5206 = true;
             TLE9201 = false;
             break;
-        case B000110:
+        case B000110: //  v 5
+            pinCheck &= B000110; // 110 for 6 
+            TB6643 = true;
+            TLE5206 = false;
+            TLE9201 = false;
+            break;
+        case B000111:  //v 1.6 AND V 1.7 (1.7 is a place holder and will not exist because 1.6 uses the same pins)
```

Comment:
Why can't 1.7 exist?

Reply:
I'm following orob lead here, i know he is testing a new TLE9201 shield and contemplating on encoding it as v1.6.
Going from base 1 to 0 encoding there is no way to not lose a #version, if we make the change from pinCheck <7 instead 8 we could have v1.7 shield but not v1.6

Reply:
Gotcha that makes sense